### PR TITLE
Show correct query result when switching history back/forward

### PR DESCRIPTION
**Pull request by simozhan**
PDC-1402

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -333,6 +390,9 @@ var Pager = React.createClass({
         if (this.props.count == 0) {
             return <div />;
         }
+        if (!this.props.showresult) {
+            return <div />;
+        }
         var n_pages = Math.ceil(this.props.count / PAGE_SIZE);
```
Comment (ycheng-aa) [inline src/app.jsx]: Could above 2 conditions be:

```
if (this.props.count == 0 or !this.props.showresult) {
    return <div />;
}
```

**Comment by simozhan** [comment]
@ycheng-aa thanks. fixed

**Comment by ycheng-aa** [comment]
:+1:

**Comment by bliuredhat** [comment]
looks good for me

**Pull request merged**